ocaml/xapi/db_gc.ml

```diff
@@ -30,7 +30,7 @@ let use_host_heartbeat_for_liveness = ref true
 
 let use_host_heartbeat_for_liveness_m = Mutex.create ()
 
-let host_heartbeat_table : (API.ref_host, float) Hashtbl.t = Hashtbl.create 16
+let host_heartbeat_table : (API.ref_host, Mtime.t) Hashtbl.t = Hashtbl.create 16
```

Comment:
I suggest against using raw monotonic timestamps because they are actually a difference between two points in time, like time spans, but they are supposed to have an implicit shared starting point. Meanwhile, there's nothing that enforces this property.

While this is not made explicit in the official documentation, it's the reason why it recommends using counters whenever possible:

> Only use [Mtime_clock.now](file:///home/paus/.local/share/opam/toolstack/var/cache/odig/html/mtime/Mtime_clock/index.html#val-now) if you need inter-process time correlation, otherwise prefer [Mtime_clock.elapsed](file:///home/paus/.local/share/opam/toolstack/var/cache/odig/html/mtime/Mtime_clock/index.html#val-elapsed) and [counters](file:///home/paus/.local/share/opam/toolstack/var/cache/odig/html/mtime/Mtime_clock/index.html#counters).


Instead, use a hashtable of counter, which contain the starting point as a reference:
```suggestion
let host_heartbeat_table : (API.ref_host, Mtime_clock.counter) Hashtbl.t = Hashtbl.create 16
```

Now knowing how much time has elapsed is more obvious:
```
let elapsed = Mtime_clock.count last_seen in
```